packages/reports/addon/components/dimension-bulk-import-simple.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ *   <DimensionBulkImportSimple
+ *      @rawInput="list,of,ids"
+ *      @onSelectValues={{this.selectValues}}
+ *   />
+ */
+import Component from '@ember/component';
+import { action } from '@ember/object';
+import layout from '../templates/components/dimension-bulk-import-simple';
+import { layout as templateLayout, tagName } from '@ember-decorators/component';
+
+@tagName('')
+@templateLayout(layout)
+class DimensionBulkImportSimpleComponent extends Component {
+  unsplitValue = [];
+  splitValues = [];
+
+  /**
+   * Filters the input list to a unique list of nonempty values
+   * @param {Array<String>} list - the list to filter
+   */
+  listFilter(list) {
+    const filter = list
+      .map(v => v.trim()) // remove surrounding space
+      .filter(v => v.length > 0); // remove empty values
+    return [...new Set(filter)]; // only unique falues
+  }
+
+  /**
+   * @method didReceiveAttrs
+   * @override
+   */
+  didReceiveAttrs() {
```

Comment:
Since this hook isn't in the glimmer API, can we use getters instead?

Reply:
It's effectively the same as using `{{did-update}}` render modifier so I switched it to that since this isn't actually a glimmer component so there's no `tracking` on the `rawInput` and thus no re-rendering